plugins/Morpheus/stylesheets/theme-advanced.less

```diff
@@ -0,0 +1,17 @@
+@theme-color-background-base:          #fff;
+@theme-color-background-tinyContrast:  #f2f2f2;
+@theme-color-background-lowContrast:   @color-silver-l80;
+@theme-color-background-contrast:      #5F5A60;
+@theme-color-background-highContrast:  #202020;
+
+@theme-color-border: @color-silver-l80;
+
+@theme-color-menu-contrast-text: @theme-color-text-lighter;
```

Comment:
OK, last time though you argued that most themes are about replacing the main Piwik colors with the one of the company… Anyway I don't mind moving those variables.